omnibus/files/private-chef-cookbooks/private-chef/recipes/default.rb

```diff
@@ -230,36 +238,41 @@
   "nginx",
   "keepalived"
 ].each do |service|
-  if node["private_chef"][service]["enable"]
-    include_recipe "private-chef::#{service}"
+  if node["private_chef"][service]["external"]
+    begin
+      # Perform any necessary configuration of the external service:
+      include_recipe "private-chef::#{service}-external"
```

Comment:
My concern with this organization is that it makes it hard to see what is different in the case of external vs non-external if we aren't very careful about how we write these cookbooks.  I'd like support to be able to ask questions like "What are the differences between internal and external postgresql?" and have the answer being easily attainable by scanning the code.